controller/search.go

```diff
@@ -1,7 +1,9 @@
 package controller
 
 import (
+	"context"
 	"fmt"
+	"github.com/fabric8-services/fabric8-wit/ptr"
```

Comment:
Grouping imports would be nice. https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
Fixed e3d4456.